addons/payment_xendit/tests/test_payment_transaction.py

```diff
@@ -12,6 +12,30 @@
 @tagged('post_install', '-at_install')
 class TestPaymentTransaction(PaymentHttpCommon, XenditCommon):
 
+    def test_no_item_missing_from_rendering_values(self):
+        """ Test that the when redirect flow is triggered, rendering_values contain the
+        API_URL corresponding to the response of API request. """
+        tx = self._create_transaction(flow='redirect')
```

Comment:
`flow` is a positional argument.
```suggestion
        tx = self._create_transaction('redirect')
```